graylog2-web-interface/src/components/navigation/Navigation.jsx

```diff
@@ -31,10 +28,12 @@ import StyledNavbar from './Navigation.styles';
 const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
 const _isActive = (requestPath, prefix) => {
+  // eslint-disable-next-line import/no-named-as-default-member
```

Comment:
Why add the `eslint-disable` instead of destructing `URLUtils`?

Reply:
To keep the change for the backport as small as possible.